cmd/lakectl/cmd/branch.go

```diff
@@ -133,11 +136,16 @@ var branchRevertCmd = &cobra.Command{
 		u := uri.Must(uri.Parse(args[0]))
 		commitRef := args[1]
 		clt := getClient()
+		hasParentNumber := cmd.Flags().Changed(ParentNumberFlagName)
+		parentNumber, _ := cmd.Flags().GetInt(ParentNumberFlagName)
+		if hasParentNumber && parentNumber <= 0 {
+			Die("parent number must be non-negative, if specified", 1)
+		}
```

Comment:
Why do you need Changed? we only validate it is non-negative value and the default is zero, which tells the server no selection was made.

Reply:
@nopcoder "changed" means that you explicitly specified it. If you do so, zero is not valid